app/controllers/notes_controller.rb

```diff
@@ -55,10 +55,56 @@ def destroy
       track_destroyed(@note)
       redirect_to node_path(@node), notice: 'Note deleted'
     else
-      redirect_to node_note_path(@node, @note), alert: 'Could not delete node'
+      redirect_to node_note_path(@node, @note), alert: 'Could not delete note'
     end
   end
 
+  def multi_destroy
+    @notes = Note.where(
+      'node_id = :node AND id in (:ids)',
+      node: @node,
+      ids: params[:ids]
+    )
+
+    if @notes.any?
+      @job_logger = Log.new(uid: (Log.maximum(:uid) || 0) + 1)
+
+      if @notes.count > Note::MAX_DELETED_INLINE
```

Comment:
I think this should be a Configuration value, so users can adjust it (some will have machines powerful enough to delete 50 at a time, some will stick to 15).

Also, we need to think if we want a new "MAX_DELETED_INLINE" setting for each model or a single one that we can use when deleting Issues, Notes, etc.